modules/cnf-du-partitioning-management-workloads.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-partitioning-management-workloads_{context}"]
+
+= Partitioning management workloads
+
+You can isolate the platform services, housekeeping tasks, and infrastructure pods to run on a reserved
+set of CPUs. This is useful for resource-constrained environments, such as single-node production deployments,
+where you want to reserve most of the CPU resources for user workloads and configure OpenShift to run on a fixed
+number of CPUs within the host.
+
+Server resources deployed at the edge, such as cores, are expensive and limited. Customers require nearly all cores for their application workloads. The resources consumed by infrastructure is a key portion of the customer's business
+case for the selection of a vRAN infrastructure. A hypothetical DU example is an unusually resource-intensive workload, typically requiring 20 dedicated cores.
+
+This feature implements the necessary OCP features and cluster configuration to isolate all housekeeping tasks and infrastructure pods to a partitioned set of CPU resources on a Single Node OpenShift (SNO) cluster with a DU profile applied. This falls broadly into two categories:
+
+* Isolates cluster management functions to the defined number of CPUs. All housekeeping tasks and infrastructure pods operate solely on that cpuset.
```

Comment:
Partitioning "cluster management" workloads from customer workloads seems to be the gist of this work.  I believe you can replace the second sentence with "All cluster management functions operate..."

"cpuset" suggests that it is a specific term but it has not been introduced yet.  It might not harm customer understanding, but it certainly isn't helping the reader might not know what a cpuset is.

Reply:
Comment accepted.